quantum/action_tapping.c

```diff
@@ -126,7 +126,7 @@ bool process_tapping(keyrecord_t *keyp) {
 #        ifdef RETRO_TAPPING_PER_KEY
                 get_retro_tapping(tapping_keycode, keyp) &&
 #        endif
-                (RETRO_SHIFT + 0) != 0 && TIMER_DIFF_16(event.time, tapping_key.event.time) < (RETRO_SHIFT + 0)
+                ((RETRO_SHIFT + 0) == 0 || TIMER_DIFF_16(event.time, tapping_key.event.time) < (RETRO_SHIFT + 0))
```

Comment:
```suggestion
                // if RETRO_SHIFT is not defined with an explicit value (`#define RETRO_SHIFT`) OR
                // the time that separates this key event from the tapping key event is below the retro shift term defined by the user (`#define RETRO_SHIFT <value>`)
                ((RETRO_SHIFT + 0) == 0 || TIMER_DIFF_16(event.time, tapping_key.event.time) < (RETRO_SHIFT + 0))
```
It isn't immediately obvious what this line means so adding a comment can help future readers from asking themselves the same question as me regarding the `+ 0`.

Although, I'm worried that the wording "the time that separates *this key event* from the *tapping key event*" may mislead some readers into thinking that `keyp` (from which `event` is derived) is necessarily a different key from `tapping_key`.

Reply:
Is that good? Not trying to explain what variables or events are in this scenario makes the most sense to me, this section just makes the `if`s with a `WITHIN_TAPPING_TERM` check also reference `RETRO_SHIFT` and this points that out.

Reply:
I would delete the two last comment lines. "[T]his [...] eliminates the hold action for use with the mouse" is kind of saying the opposite of what you mean.